data/monster/monsters/mazzinor.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Mazzinor" nameDescription="Mazzinor" race="blood" experience="100000" speed="250">
+	<health now="300000" max="300000" />
+	<look type="1062" head="85" body="27" legs="0" feet="0" addons="3" corpse="25151" />
+	<targetchange interval="2000" chance="5" />
+	<flags>
+		<flag isboss="1" />
+		<flag pushable="0" />
+		<flag canpushitems="0" />
+		<flag canpushcreatures="0" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" skill="90" attack="80" />
+		<attack name="divine missile" interval="2000" chance="10" target="1" min="-135" max="-700" />
+		<attack name="berserk" interval="2000" chance="20" range="7" target="1" min="-90" max="-500" />
```

Comment:
this need to be changed to actual attack, copy from black knight

same with divine missile above